src/main/java/twilightforest/client/TFClientSetup.java

```diff
@@ -92,6 +105,24 @@ public static void clientSetup() {
        
     }
 
+    public static void armorRegistry(){
+
+        //ArmorRenderer.register();
+
+        ArcticArmorProvider PROVIDERTEST = new ArcticArmorProvider();
```

Comment:
Change these names to something readable

Reply:
Um that's the outdated version, The providers are all based off a item from the armor set now